extensions/theme-seti/icons/vs-seti-icon-theme.json

```diff
@@ -1518,8 +1518,14 @@
 		"scmp_dark": {
 			"iconPath": "./images/scmp_inverse.svg"
 		},
-		"table-designer": {
-			"iconPath": "./images/table.svg"
+		"table-designer-temporal": {
```

Comment:
why are we removing the base icon? we still need it.

Reply:
its renaming it

Reply:
there should be 4 icons for table designer, the one we have currently (the default one), temporal, node, edge.